pandas/_libs/src/ujson/python/objToJSON.c

```diff
@@ -149,6 +149,9 @@ static TypeContext *createTypeContext(void) {
     TypeContext *pc;
 
     pc = PyObject_Malloc(sizeof(TypeContext));
+    if (pc == NULL) {
+        return NULL;
```

Comment:
Same comment here about `PyErr_NoMemory`

Reply:
Though again this is the same as L155, though I think the `pc == NULL` check is a lot more explicit